src/components/SearchElement/SearchAirlineElement/SearchAirlineElement.js

```diff
@@ -0,0 +1,71 @@
+import React, {useState, useSelector} from 'react';
+import Search from '../Search/Search';
+import SearchInstructions from '../../Text/SearchInstructions';
+import ButtonBordered from '../../UI/ButtonBordered/ButtonBordered';
+
+const  SearchAirlineElement = (props) => {
```

Comment:
This component lacks style. Define it in SearchAirlinesElement.module.css.

Reply:
Done.